ledfx/nowplaying/models.py

```
@@ -0,0 +1,91 @@
+from __future__ import annotations
+
+import hashlib
+import time
```

Flake8 (7.3.0)

[error] 'time' imported but unused

(F401)